.gitignore

```diff
@@ -11,3 +11,6 @@ obj
 docs/extensions
 .ipynb_checkpoints
 pytket/qir/_metadata.py
+test_pytket_qir_qasm_long.qasm
```

Comment:
See my other comment; if these files are useful for (local) testing they should be checked in.

Reply:
OK, then I think we should remove all references to it. If the only thing it's testing is performance then we can just move it somewhere else; if it's testing any functionality as well then we should construct another (smaller) file to test that.

Reply:
I have removed the tests, they were only for benchmarking.
See @cqc-melf
[remove qasm tests without file](https://github.com/CQCL/pytket-qir/pull/147/commits/58e65966c7cd54d03f3cc787d784228a0c6b3861)
[58e6596](https://github.com/CQCL/pytket-qir/pull/147/commits/58e65966c7cd54d03f3cc787d784228a0c6b3861)